bin/agendash-standalone-fastify.js

```diff
@@ -45,7 +49,14 @@ if (!program.path.startsWith("/")) {
 
 const fastify = Fastify();
 
-const agenda = new Agenda().database(program.db, program.collection);
+let databaseOptions = {};
```

Comment:
This can probably be a `const`, though I'm surprised the linter didn't catch it (same in the other places).